doc/releases/quincy.rst

```diff
@@ -0,0 +1,3656 @@
+======
+Quincy
+======
+
+Quincy is the 17th stable release of Ceph.  It is named after Squidward
+Quincy Tentacles from Spongebob Squarepants.
+
+v17.2.0 Quincy
+==============
+
+This is the first stable release of Ceph Quincy.
+
+Major Changes from Pacific
+--------------------------
+
+General
+~~~~~~~
+
+* Filestore has been deprecated in Quincy. BlueStore is Ceph's default object
+  store.
+
+* The `ceph-mgr-modules-core` debian package no longer recommends
+  `ceph-mgr-rook`. `ceph-mgr-rook` depends on `python3-numpy`, which
+  cannot be imported in different Python sub-interpreters multiple times
+  when the version of `python3-numpy` is older than 1.19. Because
+  `apt-get` installs the `Recommends` packages by default, `ceph-mgr-rook`
+  was always installed along with the `ceph-mgr` debian package as an
+  indirect dependency. If your workflow depends on this behavior, you
+  might want to install `ceph-mgr-rook` separately.
+
+* The ``device_health_metrics`` pool has been renamed ``.mgr``. It is now
+  used as a common store for all ``ceph-mgr`` modules.
```

Comment:
I suggest to make it clear that the pool will be renamed even on existing clusters, not just for new ones.

Reply:
done